integration/drafts_test.go

```diff
@@ -18,27 +18,27 @@ func testDrafts(t *testing.T, context spec.G, it spec.S) {
 		Expect = NewWithT(t).Expect
 	)
 
-	context("registry buildpack loader", func() {
-		it("fetches buildpack.toml from a remote buildpack", func() {
-			bp, err := drafts.RegistryBuildpackLoader{}.LoadBuildpack("gcr.io/paketo-buildpacks/bellsoft-liberica:latest")
-			Expect(err).ToNot(HaveOccurred())
-
-			Expect(bp.Info.ID).To(Equal("paketo-buildpacks/bellsoft-liberica"))
-			Expect(bp.Dependencies).ToNot(BeEmpty())
-			Expect(bp.OrderGroups).To(BeEmpty())
-			Expect(bp.Stacks).ToNot(BeEmpty())
-		})
-
-		it("fails fetching an image that does not exist", func() {
-			_, err := drafts.RegistryBuildpackLoader{}.LoadBuildpack("lasjdflaksdjfl")
-			Expect(err).To(HaveOccurred())
-			Expect(err).To(MatchError(HavePrefix("unable to load lasjdflaksdjfl")))
-		})
-	})
+	// context("registry buildpack loader", func() {
```

Comment:
commented out? why?

Reply:
Hmm.. will have a look.. not sure

Reply:
Ah.. I see.. I thought you meant just the one line.. 

I've commented out the entire method here that used the registry buildpack loader, because I removed the code that implemented it.. it wasn't in use by any of the scripts or commands.. although it's on my list of things to revisit and fix up in a follow up PR =)